frontend/src/App.js

```diff
@@ -26,19 +27,20 @@ function App() {
   return (
     <Router>
       <div className="app">
-        <Routes>
-          <Route path="/" element={!showForm ? <LandingPage onSignIn={handleSignIn} onSignUp={handleSignUp} /> : (
+         <Routes>
+          {/* <Route path="/" element={!showForm ? <LandingPage onSignIn={handleSignIn} onSignUp={handleSignUp} /> : (
```

Comment:
you commented out this part. was it intentional?

Reply:
yes I commented it intentionally because I was working on Results page only